esperanto/lukto/perdita.md

```diff
@@ -0,0 +1 @@
+La grundo ŝajnas komforta. Vi faligi kaj sorbas vin, kiel vestaĵoj. Laŭgrade komencas atingi viajn orelojn ritmon kun la notoj de piano. En la malproksimo venas la sono de trumpeto. Estas kanto. Ĝi ŝajnas preni por ĉiam kaptita en la melcocha...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.